portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,23 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  text-align: center;
   width: 650px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#map, #mapTwo{
```

Comment:
Nit: space

Reply:
Thank you for the comments, implemented the changes!